website/docs/docs/build/incremental-microbatch.md

```diff
@@ -22,17 +22,32 @@ Refer to [Supported incremental strategies by adapter](/docs/build/incremental-s
 
 Incremental models in dbt are a [materialization](/docs/build/materializations) designed to efficiently update your data warehouse tables by only transforming and loading _new or changed data_ since the last run. Instead of reprocessing an entire dataset every time, incremental models process a smaller number of rows, and then append, update, or replace those rows in the existing table. This can significantly reduce the time and resources required for your data transformations.
 
-Microbatch incremental models make it possible to process transformations on very large time-series datasets with efficiency and resiliency. When dbt runs a microbatch model — whether for the first time, during incremental runs, or in specified backfills — it will split the processing into multiple queries (or "batches"), based on the [`event_time`](/reference/resource-configs/event-time) and `batch_size` you configure.
+Microbatch is an incremental strategy designed for large time-series datasets:
+- It complements, rather than replaces, existing incremental strategies by focusing on efficiency and simplicity in batch processing.
+- Unlike traditional incremental strategies, microbatch doesn't require managing uniqueness constraints or implementing complex conditional logic for [backfilling](#backfills). 
+- It relies solely on a time column ([`event_time`](/reference/resource-configs/event-time)) to handle data partitioning and filtering. 
```

Comment:
`event_time` is technically unique from `partition_by`

also, you set `event_time` column for your microbatch model and its direct parents (might be worth calling out?)